azurerm/internal/services/containers/resource_arm_kubernetes_cluster.go

```diff
@@ -226,6 +226,18 @@ func resourceArmKubernetesCluster() *schema.Resource {
 							}, true),
 							DiffSuppressFunc: suppress.CaseDifference,
 						},
+
+						"outbound_type": {
```

Comment:
can we add [an acceptance test](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/internal/services/containers/tests/resource_arm_kubernetes_cluster_network_test.go#L12-L35) covering this new field? could we also [add this to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/kubernetes_cluster.html.markdown#L264-L288)?